test/baselines/releaseScript/README.md

```diff
@@ -0,0 +1,138 @@
+|-------------|-------------|
+| `build:ts` | Compile the TypeScript sources. |
+| `watch:ts` | Compile the TypeScript sources and watch for changes. |
+| `watch` | Shortcut for `npm run watch:ts` |
+| `test:ts` | Executes the tests you defined in `*.test.ts` files. |
+| `test:package` | Ensures your `package.json` and `io-package.json` are valid. |
+| `test:unit` | Tests the adapter startup with unit tests (fast, but might require module mocks to work). |
+| `test:integration` | Tests the adapter startup with an actual instance of ioBroker. |
+| `test` | Performs a minimal test run on package files and your tests. |
+| `check` | Performs a type-check on your code (without compiling anything). |
+| `lint` | Runs `ESLint` to check your code for formatting errors and potential bugs. |
+| `release` | Creates a new release, see [@alcalzone/release-script](https://github.com/AlCalzone/release-script#usage) for more details. |
+
+### Writing tests
+When done right, testing code is invaluable, because it gives you the 
+confidence to change your code while knowing exactly if and when 
+something breaks. A good read on the topic of test-driven development 
+is https://hackernoon.com/introduction-to-test-driven-development-tdd-61a13bc92d92. 
+Although writing tests before the code might seem strange at first, but it has very 
+clear upsides.
+
+The template provides you with basic tests for the adapter startup and package files.
+It is recommended that you add your own tests into the mix.
+
+### Publishing the adapter
+Since you have chosen GitHub Actions as your CI service, you can 
+enable automatic releases on npm whenever you push a new git tag that matches the form 
+`v<major>.<minor>.<patch>`. The necessary steps are described in `.github/workflows/test-and-release.yml`.
+
+Since you installed the release script, you can create a new
+release simply by calling:
+```bash
+npm run release
+```
+Additional command line options for the release script are explained in 
+[AlCalzone/release-script](https://github.com/AlCalzone/release-script#command-line).
+
+To get your adapter released in ioBroker, please refer to the documentation 
+of [ioBroker.repositories](https://github.com/ioBroker/ioBroker.repositories#requirements-for-adapter-to-get-added-to-the-latest-repository).
+
+### Test the adapter manually on a local ioBroker installation
+In order to install the adapter locally without publishing, the following steps are recommended:
+1. Create a tarball from your dev directory:  
+	```bash
+	npm pack
+	```
+1. Upload the resulting file to your ioBroker host
+1. Install it locally (The paths are different on Windows):
+	```bash
+	cd /opt/iobroker
+	npm i /path/to/tarball.tgz
+	```
+
+For later updates, the above procedure is not necessary. Just do the following:
+1. Overwrite the changed files in the adapter directory (`/opt/iobroker/node_modules/iobroker.test-adapter`)
+1. Execute `iobroker upload test-adapter` on the ioBroker host
```

Comment:
By the way - we should replace this with the dev-server in a separate PR.
Or at least mention it as the preferred alternative and keep this as the "fallback" manual solution.

Reply:
wilco